tasty-script-exitcode/tasty-script-exitcode.cabal

```diff
@@ -29,13 +29,13 @@ library
   exposed-modules:     Test.Tasty.HUnit.ScriptExit
   -- other-modules:
   other-extensions:    NamedFieldPuns
-  build-depends:       base >=4.12 && <4.14,
-                       process ^>=1.6.3.0,
-                       filepath ^>=1.4.2.1,
-                       directory ^>=1.3.3.0,
-                       containers ^>=0.6.0.1,
-                       tasty-hunit ^>= 0.10,
-                       tasty ^>= 1.2,
-                       temporary ^>= 1.3
+  build-depends:       base >=4.12,
```

Comment:
The purpose of the `^>=` is to add a PVP-based upper bound. This change asserts that we'll work with all the future versions of these packages - is that what we want to be saying?

Reply:
I'm not sure that I agree that that's what it means. In my opinion it means that we don't know that we _don't_ work with future versions and don't want to rule them out until we do know that they'l cause problems.

Reply:
I was going off the descriptions here: https://cabal.readthedocs.io/en/3.4/cabal-package.html#build-information

But the Platonic form of meaning of the operators and their realization in practice is certainly different, and in practice, we'll have bounds imposed by Cryptol itself anyway.